ui/apps/platform/cypress/fixtures/integrations/externalBackups.json

```diff
@@ -23,6 +23,29 @@
                 "endpoint": "test"
             }
         },
+        {
+            "id": "05fea766-e2f8-44b3-9959-eaa61a4f7466",
+            "name": "S3 API Compatible Test",
+            "type": "s3compatible",
+            "schedule": {
+                "intervalType": "UNSET",
+                "hour": 0,
+                "minute": 0,
+                "weekly": {
+                    "day": 0
+                }
+            },
+            "backupsToKeep": 0,
+            "s3compatible": {
+                "bucket": "test",
+                "accessKeyId": "******",
+                "secretAccessKey": "******",
+                "region": "test",
+                "objectPrefix": "test",
+                "endpoint": "test",
+                "urlStyle": "S3_URL_STYLE_PATH"
+            }
+        },
```

Comment:
@stehessel Considering @lvalerom  earlier comment, does it make sense to modify this file? I searched for references to this file and didn't find any. We can try undoing the changes to this file and see if everything still works as intended.

Reply:
I'll merge this now to keep the file consistent. But I'll create another PR to remove the files together with the other presumably unused fixtures.